packages/oui-dual-list/src/source/dual-list-source.html

```diff
@@ -0,0 +1,82 @@
+<!-- Header -->
+<header class="oui-dual-list__header">
+    <span class="oui-dual-list__heading" role="heading">
+        <span ng-bind="::$ctrl.heading"></span>
+        <span ng-if="!$ctrl.isLoading()">({{$ctrl.dualList.source.length}})</span>
```

Comment:
```suggestion
        <span ng-if="!$ctrl.isLoading()">(
          <span ng-bind="$ctrl.dualList.source.length"></span>
          )</span>
```

Reply:
It's way more unreadable. I'm not a fan of useless span for a simple bind.
There's no need for perf optimization here :)